src/commonMain/kotlin/kotools/types/text/NotBlankString.kt

```diff
@@ -72,18 +75,21 @@ public operator fun NotBlankString.plus(other: String): NotBlankString = "$this"
 /** Concatenates this string with the [other] one. */
 @ExperimentalSinceKotoolsTypes("4.2")
 @ExperimentalTextApi
+@JvmName("plusOnAndWithNotBlankString")
 public operator fun NotBlankString.plus(other: NotBlankString): NotBlankString =
     plus("$other")
```

Comment:
This function could include an example in its documentation for calling it from Java code.